src/Marketplace/MarketplaceScreen.js

```diff
@@ -22,9 +30,21 @@ const styles = StyleSheet.create({
     marginTop: 10,
     backgroundColor: '#0492c2',
   },
+  card: {
+    width: '90%',
+  },
 });
 
 export default function MarketplaceScreen() {
```

Comment:
Please put the navigation prop back. You can have some random button that navigates somewhere to deal w the not used error or you can also put the profile button back. You can refer here to see the original file if you're unable to revert back https://github.com/lablueprint/apifm-mobile-app/blob/master/src/Marketplace/MarketplaceScreen.js